p/promethus_clientmodel/prometheus_ubi_8.5.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : prometheus
+# Version       : v0.2.0
+# Source repo   : https://github.com/prometheus/client_model
+# Tested on     : UBI: 8.5
+# Language      : Go
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Shreya Kajbaje <shreya.kajbaje@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+PACKAGE_NAME=github.com/prometheus/client_model
+PACKAGE_VERSION=${1:-v0.2.0}
+
+set -e
+
```

Comment:
No need to add set -e here. Since it's already added as an argument to `#!/bin/bash`